includes/classes/class-db-driver-elasticpress.php

```diff
@@ -1,6 +1,9 @@
 <?php
+
 namespace ElasticPress\Stream\Driver;
 
+use ElasticPress\Elasticsearch as ElasticSearch;
```

Comment:
Different from WordPress and ElasticPress, Elasticsearch is really spelled as `Elasticsearch` with a lower case `s`, so we do not need to alias here.

```suggestion
use ElasticPress\Elasticsearch;
```